R/mod_ParticipantDomain_Server.R

```diff
@@ -32,7 +37,8 @@ mod_ParticipantDomain_Server <- function(id, rctv_lData, rctv_strName) {
           out_gtInteractive(selection_mode = "multiple") %>%
           gt::cols_label(
             .list = gsm::MakeParamLabelsList(colnames(df))
-          )
+          ) %>%
+          out_gtSmartFmtNumbers(intMaxDecimals = 10L)
```

Comment:
Nothing to change now, but should we ask users about the degree of precision they'll need here, or just default to "highly-precise"?

Reply:
Yeah, I'm definitely thinking we'll want to tune this eventually. The example I gave ends up rounding to 2 decimal places, but I'm not sure what real data will look like. I figured starting with too much and then decreasing if anything is bad is better than cutting off data.

Reply:
participant labs was getting rounded to 10, so thats what made me poke around at what the defaults were. nbd, but maybe something to think about once real data is being used as you said.
![image](https://github.com/user-attachments/assets/2d559905-6de3-44b3-8d26-67a8bdc58d05)

Reply:
Doh! That's the report I was specifically looking at, but evidently not that site! Interesting!